runtime/j9vm/java11vmi.c

```diff
@@ -1812,7 +1812,24 @@ JVM_GetNestMembers(JNIEnv *env, jclass clz)
 JNIEXPORT jboolean JNICALL
 JVM_AreNestMates(JNIEnv *env, jclass clzOne, jclass clzTwo)
 {
-	assert(!"JVM_AreNestMates unimplemented");
+	j9object_t clazzOne = NULL;
+	j9object_t clazzTwo = NULL;
+
+	if ((NULL != clzOne) && (NULL != clzTwo)) {
+		clazzOne = J9_JNI_UNWRAP_REFERENCE(clzOne);
+		clazzTwo = J9_JNI_UNWRAP_REFERENCE(clzTwo);
+
+		if (clazzOne == clazzTwo) {
```

Comment:
Can this `equivalent` check be moved before invoking `J9_JNI_UNWRAP_REFERENCE`?

Reply:
JNI references will be pointers to the Java stack, so even if they are the same object it will not return true